components/NavigationBar/src/MDCNavigationBar.m

```diff
@@ -278,16 +278,38 @@ - (NSInteger)indexOfAccessibilityElement:(id)element {
 - (void)layoutSubviews {
   [super layoutSubviews];
 
+  UIEdgeInsets RTLFriendlySafeAreaInsets;
```

Comment:
UIEdgeInsets is a struct not an id, so we need to initialize.
UIEdgeInsets RTLFriendlySafeAreaInsets = UIEdgeInsetsZero;